espflash/src/cli/mod.rs

```diff
@@ -39,7 +39,7 @@ pub mod monitor;
 mod serial;
 
 /// Establish a connection with a target device
-#[derive(Debug, Args)]
+#[derive(Default, Debug, Args)]
```

Comment:
I don't think we want to derive `Default` here. If a there are sensible defaults for specific fields, like `baud`, we should implement Default to set those fields specifically.